### PR TITLE
OCPBUGS-31380: Add prerequisite to IBU rollback to check for expired certs

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.16, 4.17

Issue: https://issues.redhat.com/browse/OCPBUGS-31380

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by achuzhoy — APPROVED** [review]

File: modules/cnf-image-based-upgrade-rollback.adoc [hunk 1]
```diff
@@ -30,6 +30,7 @@ You can manually roll back the changes if you encounter unresolvable issues afte
 .Prerequisites
 
 * Log in to the hub cluster as a user with `cluster-admin` privileges.
+* Ensure that the control plane certificates on the original stateroot are valid. If the certificates expired, see _Recovering from expired control plane certificates_.
```
Comment (aireilly) [inline modules/cnf-image-based-upgrade-rollback.adoc:33]:
```suggestion
* Ensure that the control plane certificates on the original stateroot are valid. If the certificates expired, see "Recovering from expired control plane certificates".
```
https://www.ibm.com/docs/en/ibm-style?topic=references-citations#general-guidelines

> If you refer to a smaller piece of work within a larger container such as a chapter within a book or an article in a magazine, use italic font for the larger container and double quotation marks for the smaller piece.

TL/DR double quote for referring to other docs sections

File: modules/ztp-image-based-upgrade-rollback.adoc [hunk 2]
```diff
@@ -7,6 +7,10 @@
 
 If you encounter an issue after upgrade, you can start a manual rollback.
 
+.Prerequisites
+
+* Ensure that the control plane certificates on the original stateroot are valid. If the certificates expired, see _Recovering from expired control plane certificates_.
```
Comment (aireilly) [inline modules/ztp-image-based-upgrade-rollback.adoc:12]: Same comment as above.

**Review by aireilly — COMMENTED** [review]
Looks good! Added two small comments.

**Pull request merged**